src/controllers/authController.js

```diff
@@ -5,6 +5,30 @@ const CustomError = require('../errors');
 const { attachCookiesToResponse, createTokenUser } = require('../utils');
 
 const register = async (req, res) => {
+  /* 
+    #swagger.summary = 'Register a new user'
+    #swagger.parameters['user'] = {
+      in: 'body',
+      description: 'User registration information',
+      required: true,
+      schema: {
+        $name: "John Doe",
+        $email: "jdoe@example.com",
+        $password: "supersecretpassword",
+        $username: "jdoe99"
+      }
+    }
+    #swagger.responses[201] = {
+      description: 'User registered successfully',
+      schema: {
+        $ref: '#/definitions/TokenizedUser'
+      }
+    }
+    #swagger.responses[400] = {
+      description: 'Bad request, missing email, name, password, or username'
+    }
+  */
+
```

Comment:
I think you're running into a similar bug with the swagger-autogen library as @NatachaKey  did when working on the user controller docs. I'm still not yet sure why this happens and I'll be submitting a bug report to the library repo. But in the meanwhile, it seems to be fixed when you move the swagger doc comments to the _bottom_ of the function istead of the top (for example look at what was done for usercontroller here: [https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/PS111-API-docs-auth-controller/src/controllers/userController.js#L27-L48)](https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/PS111-API-docs-auth-controller/src/controllers/userController.js#L27-L48))

It only seems to be an issue on the login and register handlers right now but you can move logout comments to the bottom too if you prefer. 

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/5568aab5-13ee-470f-be24-08c67b98a0d4)